x-pack/plugins/monitoring/server/lib/logstash/get_paginated_pipelines.ts

```diff
@@ -131,32 +127,30 @@ function processPipelinesAPIResponse(
 async function getPaginatedThroughputData(
   pipelines: Pipeline[],
   req: LegacyRequest,
-  lsIndexPattern: string,
   throughputMetric: PipelineThroughputMetricKey
 ): Promise<Pipeline[]> {
+  const dataset = 'node_stats';
+  const moduleType = 'logstash';
   const metricSeriesData: any = Object.values(
     await Promise.all(
       pipelines.map((pipeline) => {
         return new Promise(async (resolve, reject) => {
           try {
             const data = await getMetrics(
               req,
-              lsIndexPattern,
+              moduleType,
               [throughputMetric],
               [
                 {
                   bool: {
                     should: [
+                      { term: { 'data_stream.type': 'metrics' } },
```

Comment:
Would this cause this `bool` to match all `metrics` types instead of just the desired dataset? Do we have to group this in a dedicated `filter` `bool` to prevent that?

(applies to other similar changes below as well)

Reply:
Thanks @weltenwort I removed this from the `createQuery` function but forgot to remove it in these hard coded ones.  this should not be in the `should`.  As this point I am leaving it out of the query altogether since we already have the `data_stream.dataset` value and are querying the `metrics-` index pattern.

Reply:
Not really, as long as the index names include the `metrics-` prefix :+1: (Otherwise we could run into situations where the dataset is not unique across data stream types)